client/my-sites/plans/ecommerce-trial/wooexpress-plans/index.tsx

```diff
@@ -71,9 +71,12 @@ export function WooExpressPlans( props: WooExpressPlansProps ) {
 				...yearlyControlProps,
 				content: (
 					<span>
-						{ translate( 'Pay Annually (Save %(percentageSavings)s%%)', {
-							args: { percentageSavings },
-						} ) }
+						{ translate( 'Pay Annually' ) }
+						<span className="wooexpress-plans__interval-savings">
+							{ translate( ' (Save %(percentageSavings)s%%)', {
+								args: { percentageSavings },
+							} ) }
+						</span>
```

Comment:
To keep the full context in a single translation, I think it would be better to interpolate the wrapper span.

I also think we need to wrap this in a `hasTranslation()` check given that we likely have a number of non-English users that we don't want to break this for.
```suggestion
						{ isEnglishLocale || i18n.hasTranslation( 'Pay Annually {{span}}(Save %(percentageSavings)s%%){{/span}}' )
							translate( 'Pay Annually {{span}}(Save %(percentageSavings)s%%){{/span}}', {
								args: { percentageSavings },
								components: { span: <span className="wooexpress-plans__interval-savings" /> },
							}
							: translate( 'Pay Annually (Save %(percentageSavings)s%%)', {
								args: { percentageSavings },
							} )
						}
```